src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -251,8 +251,9 @@ protected Task Stop()
         {
             if (helpParams == null) { helpParams = "Get-Help"; }
 
-            PSCommand checkHelpPSCommand = new PSCommand();
-            checkHelpPSCommand.AddScript(CheckHelpScript, useLocalScope: true).AddArgument(helpParams);
+            PSCommand checkHelpPSCommand = new PSCommand()
```

Comment:
I was about to complain about the lack of `var` here, but technically the type here is the return type of the method. So all I feel is ambivalence.

Reply:
I used PSCommand due to all the other comments about being explicit about the variable type 😁

Reply:
Wholly agree. In general we lose no explicit variable type constraint with `var` when the constructor is on the same line (`var x = new Banana()`), but here it's more nuanced because of a sort of Builder pattern scenario...